ts/store/reducers/backendStatus.ts

```diff
@@ -218,6 +220,26 @@ export const isCdcEnabledSelector = createSelector(
     backendStatus.map(bs => bs.config.cdc.enabled).getOrElse(false)
 );
 
+/**
+ * Return the remote config about the barcodes scanner.
+ * If no data is available or the local feature flag is falsy
+ * the default is considering all flags set to false.
+ */
+export const barcodesScannerConfigSelector = createSelector(
+  backendStatusSelector,
+  (backendStatus): BarcodesScannerConfig =>
+    backendStatus
+      .map(bs => bs.config.barcodesScanner)
+      // If the local feature flag is disabled all the
+      // configurations should be set as `false`.
+      .chain(barcodesScanner =>
+        scanAdditionalBarcodesEnabled ? some(barcodesScanner) : none
+      )
```

Comment:
```suggestion
      .filter(() => scanAdditionalBarcodesEnabled)
```
What do you think about it? Could it be an improvement?